test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -33,6 +33,7 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/component-base/featuregate"
 	featuregatetesting "k8s.io/component-base/featuregate/testing"
+	"k8s.io/component-base/metrics/legacyregistry"
```

Comment:
should we be migrating off the legacyregistry?

Reply:
I will leave it for sig-instrumentation to comment.

BTW: this is not a brand new dependency - scheduler_perf/util.go used to and still depends on it.